test/docfx.Test/lib/YamlUtilityTest.cs

```diff
@@ -269,6 +270,17 @@ public void TestStringEmpty()
             Assert.Null(value);
         }
 
+        [Fact]
+        public void TestDuplicatedKeys()
```

Comment:
Since you start working on this, can you take a look at #2838 ? You can ignore the `null` part of that issue. We should report an error if the input yaml contains duplicated keys.

Reply:
sure thing